tailwind.config.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
   ],
   theme: {
     extend: {
+      backgroundImage: {
+        'chapter-2-square':
```

Comment:
I don't think it makes sense to add these to `tailwind.config.js` since we're only using them once. Generally you only want to add something here if you plan on using it multiple times, otherwise its just annoying for someone if they go to the chapter and discover they have to go to the tailwind config instead. So let's move these to the chapters.